### DISCUSSION TITLE
TabPanel(TabView) with Tooltip

### DISCUSSION THREAD

**Discussion by tomsoja978** [primefaces/community#1010] [PrimeReact] [answered]
I am trying to set up a Tooltip over a TabPanel but it just won't show up

https://primereact.org/tooltip/
https://primereact.org/tabview/

Can anybody please explain me what I am missing?

`<TabPanel className="heart-tooltip" data-pr-tooltip="Here is a tooltip" data-pr-position="right" leftIcon="pi pi-heart mr-0" contentClassName="heart-tooltip">`
              `<div className="flex flex-column ">`
               ` <div class="grid">{someComponents}</div>`
            ` </div>`
            `</TabPanel>`
            `<Tooltip target=".heart-tooltip" />`

**Comment by melloware** [comment, marked as answer]
Working example you have to use Passthrough attributes to put data attributes on a TabPanel:
https://stackblitz.com/edit/1ifzhc?file=src%2FApp.jsx
